exchanges/deribit/deribit_websocket.go

```diff
@@ -514,12 +514,12 @@ func (d *Deribit) processTrades(respRaw []byte, channels []string) error {
 		}
 	}
 	if tradeFeed {
-		for i := range tradeDatas {
-			d.Websocket.DataHandler <- tradeDatas[i]
+		for i := range tradesData {
+			d.Websocket.DataHandler <- tradesData[i]
 		}
```

Comment:
```suggestion
	d.Websocket.DataHandler <- tradesData
```

See discussion [here](https://github.com/thrasher-corp/gocryptotrader/pull/1777#issuecomment-2777622790)